.travis.yml

```diff
@@ -6,9 +6,11 @@ group: edge
 before_install:
```

Comment:
Your build is failing because travis has an older version of cmake.

A starting point on hot to upgrade cmake on travis-ci: https://github.com/travis-ci/travis-ci/issues/4631

You might want to use ano